auto_nag/scripts/tracked_attention.py

```diff
@@ -0,0 +1,272 @@
+            "action",
+        ]
+
+    def handle_bug(self, bug, data):
+        is_no_assignee = utils.is_no_assignee(bug["assigned_to"])
+        last_comment = self._get_last_comment(bug)
+
+        # If we commented before, we want to send reminders when we are close to
+        # the soft freeze.
+        is_reminder = bool(last_comment)
+        if is_reminder and (
+            self.is_weekend
+            or not is_no_assignee
+            or not self.show_soft_freeze_comment
+            or last_comment["time"] > self.reminder_commit_date
+        ):
+            return None
+
+        bugid = str(bug["id"])
+
+        def format_flag(flag: dict) -> str:
+            tracking_type = (
+                "tracked for" if bug[flag["tracking_field"]] == "+" else "blocking"
+            )
+            version = flag["version"]
+            channel = flag["channel"]
+            return f"{tracking_type} firefox{version} ({channel})"
+
+        tracking_statuses = [
+            format_flag(flag)
+            for flag in self.version_flags
+            if bug.get(flag["tracking_field"]) in ("blocking", "+")
+            and bug.get(flag["status_field"]) in ("affected", "---")
+        ]
+        assert tracking_statuses
+
+        reasons = []
+        if is_no_assignee:
+            reasons.append("isn't assigned")
+        if bug["severity"] in LOW_SEVERITY:
+            reasons.append("has low severity")
+        if bug["priority"] in LOW_PRIORITY:
+            reasons.append("has low priority")
+        assert reasons
+
+        # We are using the regressed_by field to identify regression instead of
+        # using the regression keyword because we want to suggesting backout. We
+        # can only suggest backout if we know the exact cause of the regression.
+        is_regression = bool(bug["regressed_by"])
+
+        # This is a workaround to pass the information to get_mail_to_auto_ni()
+        bug["is_reminder"] = is_reminder
+
+        data[bugid] = {
+            "tracking_statuses": tracking_statuses,
+            "reasons": reasons,
+            "is_regression": is_regression,
+            "action": "Reminder comment" if is_reminder else "Needinfo",
+        }
+
+        str_reasons = utils.english_list(reasons)
+        str_tracking_statuses = utils.english_list(tracking_statuses)
+        if is_reminder:
+            comment_num = last_comment["count"]
+            self.autofix_changes[bugid] = {
+                "comment": {
+                    "body": (
+                        f"This is a reminder regarding [comment #{comment_num}]"
+                        f"(https://bugzilla.mozilla.org/show_bug.cgi?id={bugid}#{comment_num})!\n\n"
+                        f"The bug is marked as { str_tracking_statuses }. "
+                        "We have limited time to fix this, "
+                        f"the soft freeze is in { self.soft_freeze_days } days. "
+                        f"However, the bug still {str_reasons}."
```

Comment:
`the bug is still`

We shouldn't remind if the only problem is that the bug has low severity

Reply:
We remind only if the bug is not assigned, not weekend, close to soft-freeze, and no recent comment from the same tool:

https://github.com/mozilla/relman-auto-nag/blob/546a3cf17e40081b6a6d2f67000deb4e70eb0a6f/auto_nag/scripts/tracked_attention.py#L92-L98